ledger/participant-state/kvutils/src/main/protobuf/daml_kvutils.proto

```diff
@@ -321,7 +324,7 @@ message DamlStateValue {
     DamlContractState contract_state = 2;
     DamlCommandDedupValue command_dedup = 3;
     DamlPartyAllocation party = 4;
-    DamlContractKeyState contract_key_state = 5;
+    string contract_key_state = 5;
```

Comment:
Please keep it as a message so we can extend it if needed.

Reply:
fixed